### PR TITLE
feat(ZENKO-1045): Add Ceph as end-to-end backend

### PR DESCRIPTION
**Pull request by tmacro**
**Which issue does this PR fix?**

fixes ZENKO-1045

NOTE: The cloudserver is set to a custom image, as it requires cloudserver PR #1662 to be merged first, same with orbit-simulator

### CODE REVIEW

File: eve/workers/ceph/chart/values.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+ceph:
+  name: ceph-in-a-box
+  image:
+    repository: docker.io/tmacro/ceph
```
Comment (ssalaues) [inline eve/workers/ceph/chart/values.yaml]: Can we push a zenko/ceph before merging?

File: eve/ci-values.yaml [hunk 2]
```diff
@@ -82,12 +82,16 @@ global:
 
 cloudserver:
   image:
+    repository: tmacro/cloudserver
```
Comment (ssalaues) [inline eve/ci-values.yaml]: With the cloudserver PR merged, we can bump this to the zenko/cloudserver.

**Review by ssalaues — COMMENTED** [review]
Just don't forget to update the images otherwise lgtm

**Review by ssalaues — APPROVED** [review]

**Comment by tmacro** [comment]
@bert-e approve

**Comment by tmacro** [comment]
@bert-e reset

**Review by philipyoo — APPROVED** [review]

**Pull request merged**